source/extensions/filters/http/transformation/inja_transformer.cc

```diff
@@ -1063,6 +1070,19 @@ void InjaTransformer::transform(Http::RequestOrResponseHeaderMap &header_map,
     }
   }
 
+  // Span transform:
+  if (span_name_template_.has_value()) {
+    // If route.decorator.operation is set, do not update the span name.
+    if (callbacks.route()
+        && callbacks.route()->decorator()
+        && !callbacks.route()->decorator()->getOperation().empty()) {
+      callbacks.activeSpan().setOperation(callbacks.route()->decorator()->getOperation());
```

Comment:
?

Reply:
TODO delete this

Reply:
This was refactored in https://github.com/solo-io/envoy-gloo/pull/368/commits/3d7463c0748ecbc6ec3dcf309a5b4fd19d9b81df